docs/source/poutine.rst

```diff
@@ -1,4 +1,9 @@
 Poutine (Effect handlers)
 ==========================
 
+.. automodule:: pyro.poutine
```

Comment:
Could you add a subsection title above this automodule, e.g.
```
Poutine Primitives
------------------
```
This is needed because, whereas all the `Handler`s appear in the TOC and the main poutine page, it's not apparent that any documentation exists for `trace`, `blocke` etc. until you click on the chapter heading.

Reply:
(take a look at the generated page. do you also find it confusing?)